.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: win
   pool:
-    vmImage: windows-2019
+    vmImage: windows-2022
```

Comment:
```suggestion
    vmImage: windows-2019
```

Reply:
If you want the image change to survive a rerender, you can add a commit like: https://github.com/conda-forge/lld-feedstock/commit/e1d563f45e24ee4084f9f3c33fd847a26c9a0f80

Reply:
Perfect. Thanks